### PR TITLE
Sr minor fixes

### PR DESCRIPTION
**Pull request by steverydz**
Some general cleanup and fixes for the hover menus

### CODE REVIEW

File: assets/js/src/script.js [hunk 1]
```diff
@@ -64,20 +64,41 @@ $(document).ready(function () {
...
 
-    ApiDocs.init(data);
+    apiDocs.init(data);
   });
 
   var apiExplorer = new ApiExplorer();
   apiExplorer.init();
+
+  // Required to make submenu's dissappear on click
+  $('.has-submenu a').not('.user-menu-link').on('click', function () {
```
Comment (yemble) [inline assets/js/src/script.js]: Can you combine these two into one handler `on('click mouseover', ..` ?

File: assets/templates/logged-in.ejs [hunk 2]
```diff
@@ -1,9 +1,9 @@
-<div class="navbar-user logged-in has-dropdown-menu">
-  <a class="user-avatar" href="#">
+<div class="navbar-user logged-in has-submenu">
+  <a class="user-menu-link" class="user-avatar" href="#">
```
Comment (yemble) [inline assets/templates/logged-in.ejs]: two class attributes

File: assets/js/src/script.js (hunk 1 above)
Comment (steverydz) [inline assets/js/src/script.js]: Not really as they're both doing different things.

File: assets/templates/logged-in.ejs (hunk 2 above)
Comment (steverydz) [inline assets/templates/logged-in.ejs]: Good catch. Fixed.

**Pull request merged**